FunctionSignatureMap.php

```diff
@@ -9859,7 +9859,7 @@
 'SessionUpdateTimestampHandler::validateId' => ['char', 'id'=>'string'],
 'SessionUpdateTimestampHandlerInterface::updateTimestamp' => ['bool', 'key'=>'string', 'val'=>'string'],
 'SessionUpdateTimestampHandlerInterface::validateId' => ['bool', 'key'=>'string'],
-'set_error_handler' => ['?string', 'error_handler'=>'null|callable(int,string,string,int,array):bool|callable(int,string,string,int):bool|callable(int,string,string):bool|callable(int,string):bool', 'error_types='=>'int'],
+'set_error_handler' => ['?callable', 'error_handler'=>'null|callable(int,string,string=,int=,array=):bool', 'error_types='=>'int'],
```

Comment:
I overlooked the changes to the parameter union type - Those will need to be changed due to differences in Phan/Psalm's casting rules for callables, as discussed earlier.

The change to the return type makes sense.

Reply:
The fix was committed to master.

Reply:
Awesome, sorry